packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -407,8 +423,12 @@ export class ComponentDecoratorHandler implements
       }
     }
 
-    let inlineStyles: string[]|null = null;
-    if (component.has('styles')) {
+    // If inline styles were preprocessed use those
+    let inlineStyles = this.preanalyzeStylesCache.get(node) || null;
```

Comment:
Even though we still downlevel to ES5 we have started to adopt `??` in compiler code:

```suggestion
    let inlineStyles = this.preanalyzeStylesCache.get(node) ?? null;
```

Reply:
I'd be nice if we could avoid the null completely here actually